### PR TITLE
impr(layouts): add basic support for thumb alpha keys (@zigotica)

### PR DESCRIPTION
**Pull request by zigotica**
### Description

ROW 5 in alternate keymaps allow for alphas in thumb keys.
* These keymaps MUST include two keys in row 5, an alpha and a space, or a space and an alpha.
* Alpha key is rendered with the regular alpha size.
* Layout name is automatically added in the space key.
* Added keymaps using alphas in thumbs: anishtro, BEAKL_Zi, MALTRON, PRSTEN and RSTHD

### Checks

- [x] Made sure all physical layouts work for alternate and classic keymaps. Attaching images for all of them in the PR.
alice alpha left
alice alpha right
alice classic
matrix alpha left
matrix alpha right
matrix classic
split-matrix alpha left
split-matrix alpha right
split-matrix classic
split alpha left
split alpha right
split classic
staggered alpha left
staggered alpha right
staggered classic
steno-split
steno

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username prefixed with @ inside parentheses at the end of the PR title.

Closes https://github.com/monkeytypegame/monkeytype/pull/2291

### CODE REVIEW

**Comment by Miodec** [comment]
Please move the layouts to the bottom of the list file.

**Comment by Miodec** [comment]
Other than those two I think its all good.

**Comment by zigotica** [comment]
> Please move the layouts to the bottom of the list file.

moved

File: frontend/src/ts/elements/keymap.ts [hunk 1]
```diff
@@ -203,19 +207,76 @@ export async function refresh(
...
+          if (keyVisualValue === " ") {
+            rowElement += `<div class="keymapKey keySpace layoutIndicator ${side}">
+              <span class="letter" ${letterStyle}>${layoutDisplay}</span>
+            </div>`;
+            r5_grid += "3";
+            // potential second space in next loop iterations will be empty:
+            layoutDisplay = "";
+          } else {
+            rowElement += `<div class="keymapKey ${side}">
+              <span class="letter">${keyDisplay}</span>
```
[the remark this answers is not in the document]
Comment (zigotica) [reply frontend/src/ts/elements/keymap.ts]: idk, sorry. changed

**Comment by Miodec** [comment]
Thanks

**Pull request merged**